_data/simple-icons.json

```diff
@@ -2757,6 +2757,11 @@
             "hex": "E7352C",
             "source": "https://www.espressif.com/"
         },
+        {
+            "title": "Etcd",
```

Comment:
I feel like their preferred way of writing the name is "etcd", not "Etcd". Would you agree? If so, could you update it here and in the SVG? If not, could you share your reasoning?

Reply:
done :)